src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1148,6 +1195,45 @@ private IEnumerable<DocumentItem> ReadDocuments(INewDocumentActions actions = nu
             }
         }
 
+        private bool TryMatchAttachmentToDocumentOrStoreForLaterMatching(
+            Dictionary<string, DocumentItem> storeDocsForLaterMatchWithAttachments,
+            Dictionary<string, DocumentItem.AttachmentStream> storeAttachmentsForLaterMatchWithDocs,
+            DocumentItem.AttachmentStream attachment,
+            bool reportOrphan = false)
+        {
+            if (attachment.Data.TryGetMember("Tag", out var tag))
```

Comment:
You cannot use this for anything. This is information only, debug, not operational